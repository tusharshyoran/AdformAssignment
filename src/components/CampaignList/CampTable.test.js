import React from "react";
import CampaignTable from "./CampTable";
import { Provider } from "react-redux";
import store from "../../redux/store";
import renderer from "react-test-renderer";

describe("Campaign List", () => {
  it("should render Campaign List correctly", () => {
    const component = renderer
      .create(
        <Provider store={store}>
          <CampaignTable />
        </Provider>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
