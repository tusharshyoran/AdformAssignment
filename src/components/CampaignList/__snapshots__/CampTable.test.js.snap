// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Campaign List should render Campaign List correctly 1`] = `
<div>
  <table
    className="table"
  >
    <thead>
      <tr>
        <th
          scope="col"
        >
          Name
        </th>
        <th
          scope="col"
        >
          User Name
        </th>
        <th
          scope="col"
        >
          Start Date
        </th>
        <th
          scope="col"
        >
          End Date
        </th>
        <th
          scope="col"
        >
          Active
        </th>
        <th
          scope="col"
        >
          Budget
        </th>
      </tr>
    </thead>
  </table>
</div>
`;
